.pytool/Plugin/UncrustifyCheck/UncrustifyCheck.py

```diff
@@ -575,26 +575,28 @@ def _process_uncrustify_results(self) -> None:
         self._formatted_file_error_count = len(formatted_files)
 
         if self._formatted_file_error_count > 0:
-            logging.error(
+            logging.warning('Uncrustify found {0} files with formatting errors'.format(self._formatted_file_error_count))
+            self._tc.LogStdError("Uncrustify found {0} files with formatting errors:\n".format(self._formatted_file_error_count))
+            logging.info(

```

Comment:
This message needs to be very obvious for people to find the page. I'm fine for it not to be error level but will it still be printed where they can see it easily (some people don't know how to open log files)?